game_eggs/steamcmd_servers/night_of_the_dead/README.md

```diff
@@ -0,0 +1,17 @@
+# Work in progress!!!!
+
+# Night of the dead
+Steam Description  
+Build defenses against nightly waves of zombies. Survive and escape the island.
+
+### Server Ports
+Night of the dead requires up to 2 ports
+
+Game query port (default 27015)
+Game client port (default 7777)
+
```

Comment:
It would be nice to include the minimum requirements here.
I think the server needs 5GB RAM and 5GB disk space (at least)